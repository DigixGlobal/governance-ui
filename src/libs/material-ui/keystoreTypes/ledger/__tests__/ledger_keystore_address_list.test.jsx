import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import LedgerKeystoreAddressList from '../ledger_keystore_address_list';

const config = {
  version: 1,
};

const ethLedger = {
  getAddress_async: () => jest.fn,
};

describe('<LedgerKeystoreAddressList />', () => {
  test('renders correctly', () => {
    const component = shallow(
      <LedgerKeystoreAddressList
        ethLedger={ethLedger}
        config={config}
        renderContainer={jest.fn}
      />,
    );

    expect(toJson(component)).toMatchSnapshot();
  });
});
