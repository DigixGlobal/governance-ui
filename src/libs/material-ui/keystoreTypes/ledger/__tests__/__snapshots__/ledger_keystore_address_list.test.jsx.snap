// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LedgerKeystoreAddressList /> renders correctly 1`] = `
<div>
  <div
    style={
      Object {
        "float": "right",
      }
    }
  >
    Ledger 
    1
  </div>
  <PaginationMenu
    currentPage={0}
    disabled={[Function]}
    handleNavigate={[Function]}
    itemsPerPage={5}
  />
</div>
`;
